docs/cli.md

```diff
@@ -10,29 +10,23 @@ license: 'This software is licensed under the Apache License version 2.'
 
 This document provides guidance on managing metadata within Apache Gravitino using the Command Line Interface (CLI). The CLI offers a terminal based alternative to using code or the REST interface for metadata management.
 
-Currently, the CLI allows users to view metadata information for metalakes, catalogs, schemas, tables, users, roles, groups, tags, topics and filesets. Future updates will expand on these capabilities.
+Currently, the CLI allows users to view metadata information for metalakes, catalogs, schemas, tables, modules, columns, users, roles, groups, tags, topics and filesets. Future updates will expand on these capabilities.
 
 ## Running the CLI
 
-You can configure an alias for the CLI for ease of use, with the following command:
-
-```bash
-alias gcli='java -jar ../../cli/build/libs/gravitino-cli-*-incubating-SNAPSHOT.jar'
-```
-
-Or you use the `gcli.sh` script found in the `clients/cli/bin/` directory to run the CLI.
+To run the Gravitino CLI, please use the `gcli.sh` script located in Gravitino's bin directory. if you define the `GRAVITINO_HOME` environment variable, which allows you to run the script from any location using `$GRAVITINO_HOME/bin/gcli.sh`.
```

Comment:
"located in the Gravitino bin directory.... If you set the `GRAVITINO_HOME` environment variable in the current shell, you can run..."

Reply:
I'd change it to 'If you define the `GRAVITINO_HOME` environment variable, you can run the script from any location using `$GRAVITINO_HOME/bin/gcli.sh`.". Please note it's "If" not "if".